### PR TITLE
#27 chore:configure testing environment

### PR DESCRIPTION
**Pull request by arthurjunior250**
**What does this PR do?**

-Configured Testing environment with Jest & Enzyme

** Description of Task to be completed?**

-All functionalities are tested before deployment.

**#### How should this be manually tested?**

-Clone the repo

-checkout to branch ch-setup-testing-#27

--Run npm run test

#### What are the relevant pivotal tracker **stories?**

-link to my pivotal tracker/tello/https://trello.com/c/wrRkUEwa

**#### Screenshots (if appropriate)**

Screenshot from 2022-04-05 17-26-27

**#### Questions:**
-N/A

### CODE REVIEW

**Comment by shemalucien** [comment]
Good job mn I think you can add dodo, achile, and other team members as if you did not add them to review your PR.
also, try to add the label to your PR

**Review by harerajo — DISMISSED** [review]

File: src/App.test.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+import React from 'react';
+
+test('renders learn react link', () => {
+  render(<App />);
```
Comment (mukunzidd) [inline src/App.test.js:5]: The coverage script is failing at this line. Check what's wrong.

Comment (mukunzidd) [inline src/App.test.js:5]: The coverage script is failing at this line. Check what's wrong.

**Review by mukunzidd — CHANGES_REQUESTED** [review]
Fix the coverage script and the PR will be ready to be merged.

**Review by Stevenkwizera06 — DISMISSED** [review]

**Review by songa1 — COMMENTED** [review]
Please check out feedbacks given by @Dodo and solve the problems in your script and in index.js file.

**Review by songa1 — DISMISSED** [review]
Tests written are passing. And the environment is set and ready.

**Review by songa1 — APPROVED** [review]

**Review by Stevenkwizera06 — APPROVED** [review]

**Review by shemalucien — APPROVED** [review]

**Review by harerajo — APPROVED** [review]

**Review by Juleshb — APPROVED** [review]

**Pull request merged**